lib/lambda_wrap/lambda_manager.rb

```diff
@@ -79,7 +79,7 @@ def publish_lambda_to_s3(local_lambda_file, bucket, key)
 		# [function_name]	The name of the lambda function.
 		# [handler]			The handler that should be executed for this lambda function.
 		# [lambda_role]		The arn of the IAM role that should be used when executing the lambda function. 
-		def deploy_lambda(bucket, key, version_id, function_name, handler, lambda_role)
+		def deploy_lambda(bucket, key, version_id, function_name, handler, lambda_role, lambda_description)

```

Comment:
Can you add a default value for parameter `lambda_description`? Probably call it `Created by LambdaWrap`. This ensures easy transition to the latest version of LambdaWrap without forcing clients to adjust their script.